configs/prod/roles/vulnerability.json

```diff
@@ -22,7 +22,8 @@
       "name": "Vulnerability viewer",
       "description": "Perform read operations on any Vulnerability resource.",
       "system": true,
-      "platform_default": false,
+      "platform_default": true,
+      "admin_default": false,
       "version": 4,
```

Comment:
The version has to be updated

Reply:
yes that is big true thank you I totally missed that

Reply:
fixed